app/serializers/async_transaction/base_serializer.rb

```diff
@@ -2,14 +2,16 @@
 
 module AsyncTransaction
   class BaseSerializer < ActiveModel::Serializer
+    attribute :status
     attribute :transaction_id
-    attribute :transaction_status
     attribute :type
 
     def id
       nil
     end
 
-    delegate :type, to: :object
```

Comment:
I understand what's going on here, but I wonder if flipping the field names for the front end will lead to confusion, especially since the coexistence of `status` and `transaction_status` columns is already kind of confusing.

Reply:
Yeah, we chatted about this.  We have two status related attrs on this model (`status` and `transaction_status`). One of them applies to the status of **our** transaction record (`status`), and the other applies to the status of the vet360 transaction in their world (`transaction_status`).

We landed on this:  https://hackmd.io/YiHJhoC_Qvm4piIJ8RQBZg#GET-v0profilestatustransaction_id

The FE is only concerned with the status of their request, which is the vet360 status, which is `transaction_status`, and to make it more friendly and straightforward to them, we landed on `status`. 

Are you saying to revert that?